ui/orbital-stats.cpp

```diff
@@ -57,7 +59,7 @@ typedef struct orbital_stats_device_usage_t {
     orbital_stats_usage_t msi;
 } orbital_stats_device_usage_t;
 
-static const orbital_stats_device_info_t devices[] = {
+static const std::array<orbital_stats_device_info_t, 10> devices = {{
```

Comment:
Just curious, would it be possible to declare this without explicitly providing the size? Maybe using `make_array`?

Reply:
`make_array` is experimental isn't it? I could add `UI_DEVICE_LIVERPOOL_ITEMS` in `ui_device_t` and use `UI_DEVICE_LIVERPOOL_ITEMS - 1`.

Reply:
Ah, right. No worries then, if the feature is experimental your patch is alright as-is.